ui/appui-react/src/appui-react/framework/FrameworkKeyboardShortcuts.ts

```diff
@@ -5,7 +5,7 @@
 /** @packageDocumentation
  * @module KeyboardShortcut
  */
-import type { FunctionKey, SpecialKey } from "@itwin/appui-abstract";
+import type { FunctionKey, SpecialKey } from "../keyboardKey/KeyboardKey";
```

Comment:
Could you run `extract-api`, this seems like a breaking change (or just invalid), since `SpecialKey` is used in a `@public` type `KeyboardShortcutProps`.

Reply:
I think tooling doesn't complain since the newly added `@internal SpecialKey` is not in the barrel file.

Reply:
Thinking through this more, I don't see an issue. `SpecialKey` is still available, just deprecated. It can continue to be used. The property in question `KeyboardShortcutProps.key` is of type `string | FunctionKey | SpecialKey` - with both `FunctionKey` and `SpecialKey` being `string enums`, the set of acceptable types is the entire set of strings. Even if we removed `FunctionKey` and `SpecialKey` from the union type, the set of acceptable types is the same.

Reply:
Oh, good catch, `SpecialKey` enum is a subset of a string.